src/pages/index.js

```diff
@@ -84,16 +85,16 @@ Promise.all([api.getUserInfo(), api.getInitialCards()])
 // =============================================================================
 
 function handleLikeClick(cardInstance) {
-  if (cardInstance.likes) {
+  if (cardInstance._likes) {
```

Comment:
`_likes`


You shouldn't have added the underscore here again

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.